Dockerfiles/midstream/run_tests.py

```diff
@@ -52,48 +78,38 @@ def test_for_extract_and_validate_bundle_image(self):
         exit_code = 0
         # check if IMAGE_TO_TEST is defined, return exit_code 102 in case it's not
         if (self.image_to_test is None):
-            print("Environment variable IMAGE_TO_TEST not set! Stopping the tests.")
-            with open(".errormessage", 'w') as error_msg:
-                print("Result code: 102 Error message: Environment variable IMAGE_TO_TEST not set!", file=error_msg)
+            logging.error("Environment variable IMAGE_TO_TEST not set! Stopping the tests.")
+            logging.error("Result code: 102 Error message: Environment variable IMAGE_TO_TEST not set!")
             exit_code = 102
             return exit_code
-        if (self.operator_dir != '/project/operator_dir/' or
-           self.operator_work_dir != '/project/test_operator_work_dir/' or
-           self.playbooks_dir != '/project/operator-test-playbooks/' or
-           self.umoci_bin_path != '/usr/local/bin/umoci' or
-           self.work_dir != '/project/output/'):
-            print("Environment variable misconfigured!")
-            with open(".errormessage", 'w') as error_msg:
-                print("Result code: 103 Error message: Environment variable was not set correctly!", file=error_msg)
-            exit_code = 103
-            return exit_code
         result = self.run_extract_operator_bundle()
-        if (result.returncode != 0):
-            print("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file .errormessage for more info." % result.returncode)
-            with open(".errormessage", "w") as error_msg:
-                print("Result code: " + str(result.returncode), "Error message: " + str(result.stderr), file=error_msg)
+        logging.info(result["stdout"])
+        if (result["return_code"] != 0):
+            logging.error("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file .errormessage for more info." % result["return_code"])
+            logging.error("Result code: " + str(result["return_code"]))
+            logging.error("Error message: " + str(result["stderr"]))
+            logging.error("Result stdout: " + str(result["stdout"]))
             exit_code = 50
             return exit_code
         result = self.run_validate_operator_bundle()
-        assert (path.exists("/project/output/validation-rc.txt"))
-        assert (path.exists("/project/output/validation-output.txt"))
-        assert (path.exists("/project/output/validation-version.txt"))
-        with open('/project/output/validation-rc.txt', 'r') as reader:
+        logging.info(result["stdout"])
+        # make sure the validation-rc , validation-output, validation-version 
+        # files have been created due to the playbook run
+        assert (path.exists(WORK_DIR+"/validation-rc.txt"))
```

Comment:
Assertions will not be logged to the the logger `logging`
Why do we need to use assert here if users cannot see it?

Reply:
assertions have been removed